movement/watch_faces/complications/stopwatch_face.h

```diff
@@ -0,0 +1,26 @@
+#ifndef STOPWATCH_FACE_H_
+#define STOPWATCH_FACE_H_
+
+#include "movement.h"
+
+typedef struct {
+    bool measuring;
```

Comment:
Suggest renaming this variable to `running`; I sense it's a more descriptive name and would add more clarity to the code.